### PR TITLE
Add Moedict-Desktop.app v1.1.1

### PR DESCRIPTION
**Pull request by aethanyc**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses. (homepage and url not match. Comment added in moedict.rb)

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by aethanyc** [comment]
@racklin The cask needs a license field. I use MPL per https://github.com/racklin/moedict-desktop instead of cc0 in https://racklin.github.io/moedict-desktop/. I hope this is correct.

**Comment by racklin** [comment]
@aethanyc Thank you for mention me about license issue of my project.
I think you use MPL license of your PR is Okay.

Moedict-desktop-app based on Mozilla XULRunner and it's MPL 2.0 license.
But databases are from https://github.com/audreyt/moedict-webkit project, and it's CC0 license.

I will modify  https://racklin.github.io/moedict-desktop/ website  from CC0 to MPL , sorry for the confusion.

File: Casks/moedict.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'moedict' do
+  version '1.1.1'
+  sha256 'c11ce7da15777d2d84c3063d1260f2b33c3e0e2e6cf02507f38354c919693ca2'
+
+  # https://github.com/racklin/moedict-desktop/ was verified as official when first introduced to the cask
+  url 'https://github.com/racklin/moedict-desktop/releases/download/1.1.1/moedict-desktop.app-1.1.1.dmg'
```
Comment (fanquake) [inline Casks/moedict.rb]: You can use #{version} in the url stanza

Comment (aethanyc) [inline Casks/moedict.rb]: Fixed. Thanks!

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in f167ba263c973575efa611a643f91e982d6283dd. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**